Tests/UI/BillingForm/Validator/PhoneNumberValidatorTests.swift

```diff
@@ -3,17 +3,38 @@ import XCTest
 
 class PhoneNumberValidatorTests: XCTestCase {
 
-    func testValidationWhenTextIsEmptyThenShouldBeTrue() {
+    func testValidationTextWithEmptyString() {
         let expectedType = BillingFormCell.phoneNumber(nil)
         let text = ""
-        let isValid = expectedType.validator.validate(text: text)
-        XCTAssertTrue(isValid)
+        let isInvalid = expectedType.validator.validate(text: text)
+        XCTAssertTrue(isInvalid)
```

Comment:
In relation to my comment : https://github.com/checkout/frames-ios/pull/338/files#r1010412180. Here the text is empty string so the expectedType.validator.validate should return false Instead of returning true